include/BcAlgTraits.h

```diff
@@ -0,0 +1,31 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2014 Sandia Corporation.                                    */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+
+#ifndef BcAlgTraits_h
+#define BcAlgTraits_h
+
+#include <stk_topology/topology.hpp>
+
+namespace sierra {
+namespace nalu {
+
+struct BcAlgTraitsHex8Quad4 {
```

Comment:
Is the plan to write out every possible combination in this header file? Or is this a placeholder that will be eventually replaced by some templates that are derived from `struct`s in `AlgTraits.h`?

Reply:
This is a placeholder. The plan is to add a templated method that instantiates each unique pair based on AlgTrait combinations.